test/suite/test_rollback_to_stable15.py

```diff
@@ -51,8 +52,15 @@ class test_rollback_to_stable15(wttest.WiredTigerTestCase):
     ]
     scenarios = make_scenarios(key_format_values, value_format_values, in_memory_values)
 
+    # Don't raise errors for these, the expectation is that the RTS verifier will
+    # run on the test output.
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.ignoreStdoutPattern('WT_VERB_RTS')
```

Comment:
Could you explain why this is necessary in some tests and not others?

Reply:
Yes, it depends on whether a test inherits from `test_rollback_to_stable01` or not. If they do, they get it for free. Some tests (like this one) inherit from `wttest.WiredTigerTestCase` (or something else) so they need to add the teardown step themselves.